doc/release-notes.md

```diff
@@ -99,6 +99,10 @@ Low-level Changes section below.
   `-limitancestorcount`, `-limitdescendantcount` and `-walletrejectlongchains`
   command line arguments.
 
+* The `gettransaction` RPC now accepts a third (boolean) argument `decode`. If
+  set to `true`, a new `decoded` field will be added to the response containing
```

Comment:
There's another one of those chars here after `decoded` and before `field`. Not a big deal, just was wondering what editor you are using that would be adding these randomly?

```
git cat-file -p a2b23686c65d7e8c7e69b7547322d2eea1cf3b44 | xxd | grep c2a0
00000f90: 6465 6460 c2a0 6669 656c 6420 7769 6c6c  ded`..field will
```

Reply:
Vim, you right it's weird